src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  font-size: 16px;
+  padding-left: 62px;
+  line-height: 22px;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  height: 70px;
+  background-size: 19px;
+  width: 100%;
+}
+
+input::placeholder {
+  color: #3d4e61;
+}
+
+input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+  background-image: url("images/Search.svg");
+  background-position: 25px 24px;
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+}
+
+.small--bar {
+  margin-top: 20px;
+  width: 100%;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.small--bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  text-shadow: none;
+}
+
+.small--bar:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+  background-image: url("images/Search.svg");
+  background-position: 13px 15px;
+  background-repeat: no-repeat;
+  background-size: 11px;
+  text-shadow: none;
+  font-weight: 900;
+}
```

Comment:
apply only those for `input--small::focus` modifier that are different from `.input::focus`